src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -716,7 +710,7 @@ public float CalculateSpeed()
             (Constants.CELL_DRAG_MULTIPLIER -
                 (Constants.CELL_SIZE_DRAG_MULTIPLIER * lengthMicrobe))) *
             (Membrane.MovementFactor -
-            (Rigidity * Constants.MEMBRANE_RIGIDITY_MOBILITY_MODIFIER));
+            (editedSpecies.MembraneRigidity * Constants.MEMBRANE_RIGIDITY_MOBILITY_MODIFIER));
```

Comment:
I'm pretty sure this doesn't work correctly because this needs to be
1) recomputed when the rigidity slider is changed
2) use the current value